### ISSUE TITLE
Creating video Cover block in mobile app results in block validation error

### ISSUE THREAD

**Issue by dcalhoun** [WordPress/gutenberg#28972]
## Description

Creating a video Cover block within the WordPress mobile app results in block validation errors when re-editing the post.

## Step-by-step reproduction instructions

1. Download (or generate) build for the latest WordPress Android.
2. In the Android app, create a post for a Simple WordPress.com site.
3. Add a Cover block.
4. Select a video for the Cover block background.
5. Publish the post.
6. Exit the post editor.
7. Edit the same post in the mobile app OR edit the same post in the web editor.

## Expected behaviour

No errors are displayed in the mobile app. No errors are displayed in the web console.

## Actual behaviour

A React Native error is displayed regarding block validation. Opening the same post in the web editor displays a block validation error in the console.

## Screenshots or screen recording (optional)

<details>
<summary>Error Screenshots</summary>

## Web
<img width="1708" alt="video-cover-block-validation-error-web" src="https://user-images.githubusercontent.com/438664/107788968-9e735a00-6d16-11eb-89e0-f13df1e5fc7a.png">

## Android
![video-cover-block-validation-error-android](https://user-images.githubusercontent.com/438664/107788861-813e8b80-6d16-11eb-840a-9336f854ea81.jpg)

</details>

## WordPress information
- WordPress version: WordPress.com
- Gutenberg version: 9.8.4
- Are all plugins except Gutenberg deactivated? Yes
- Are you using a default theme (e.g. Twenty Twenty-One)? No

## Device information
- Device: Samsung S20 5G FE
- Operating system: Android 11
- WordPress app version: `86fe5183b1cf12d9854f17de722d1978b443ddc9` of [WordPress Android](https://github.com/wordpress-mobile/WordPress-Android/commit/86fe5183b1cf12d9854f17de722d1978b443ddc9).

**Comment by hypest** [comment]
Thanks for raising the issue David! Any info whether this leads to a crash of the editor?

**Comment by dcalhoun** [comment]
@hypest no, from my testing a crash does not occur in a release build. 

**Release build:** The web editor displays a warning in the console.
**Debug build:** The RN warning is displayed in the mobile app and the web editor displays a warning in the console.

**Comment by youknowriad** [comment]
Hey folks, just asking about the status of the mobile app in Gutenberg, there's a number of issues that are related and it feels like we're moving away from it. I wonder if we should close the related issues.

**Comment by dcalhoun** [comment]
👋🏻 Hey @youknowriad. The majority of the active development related to the block editor in the WordPress mobile app is currently occurring in the [GutenbergKit repository](https://github.com/wordpress-mobile/GutenbergKit). The GutenbergKit block editor variant remains behind an experimental feature flag and, thus, is not actively promoted to or used by all mobile app users at this time. We hope to more heavily promote it early next year.

Regarding closing issues related to the React Native block editor, I think that is fine. Once GutenbergKit is more established, the ideal outcome is housing its source code and issue tracking in the Gutenberg repository.

**Comment by youknowriad** [comment]
Ok Thanks for the update. I'm going to hold off for now but let me know when you think we can go ahead and do:

 - close all issues.
 - remove native code.

**Comment by dcalhoun** [comment]
@youknowriad sounds good. I'm equally eager to clean up. 🙂

**Comment by tyxla** [comment]
This is no longer relevant after the React Native implementation was removed in #78747. Closing.
